src/components/PopularSkillsModule.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { get, flow, pick, map } from 'lodash/fp'
+import { sendGraphqlQuery } from '../actions/graphql'
+import { connectedListProps } from '../util/caching'
+import Avatar from './Avatar'
+import A from './A'
+import { peopleUrl } from '../routes'
+
+const { func, object, array } = React.PropTypes
+
+const fetchPopularSkills = slug =>
+  sendGraphqlQuery(`query ($slug: String) {
```

Comment:
@levity compare this function call with the call here:
https://github.com/Hylozoic/hylo-redux/blob/d39666a4d9865e6ff1f62ef83dd51bd8ec4e5255/src/components/PopularSkillsModule.js
the extra nesting, and the extra 'data' component of all the `get` paths (which obfuscates the relationship between the query structure and the payload structure) are why I wanted to have a separate `sendGraphqlQueryAddDataToStore` function.

I wonder if, seeing this, you still want to just use the basic `sendGraphqlQuery` here? And if so, I'd be curious to hear you're reasoning.